### DISCUSSION TITLE
CCO Connector login fails for an account using the US GOV cloud

### DISCUSSION THREAD

**Discussion by scmurra** [Azure/CCOInsights#164] [Q&A]
When logging into the CCO Dashboard using the connector, login fails for an account using the US GOV cloud.  
[DataSource.Error] Web contents failed to get contents from 'https://login.windows.net/common/oauth2/token'  (400): Bad Request

This error only occurs when loading Governance dashboard (Custom Connector).  Loading the Infrastructure dashboard works as expected.

![image](https://user-images.githubusercontent.com/50504648/113917980-e8753b00-97af-11eb-83c5-4ae3789b8c62.png)

**Comment by justinharty1** [comment]
I also am having the same issue as I work in gov cloud as well.

The only thing I can think of right now is either an authentication address is hard coded for global, or an address changed somewhere that the code didn't update.

however I did a manual test of the link login.windows.net, and based on my login credentials, it did alot of redirection back and forth and eventually came back to office365.us (gcc), so...something in the link line itself that is appended to it...?

and likewise, the infrastructure view was able to connect.

**Reply by lsutigers1162** [reply]
has anyone figured out how to run these reports against multiple tenants with different login credentials to each tenant?
